package.json

```diff
@@ -59,7 +60,8 @@
     "yarn": "^1.22.21"
   },
   "resolutions": {
-    "@coinbase/wallet-sdk": "npm:@coinbase/wallet-sdk@4.0.0"
+    "@coinbase/wallet-sdk": "npm:@coinbase/wallet-sdk@4.0.0",
+    "react": "npm:react@^18"
```

Comment:
oh interesting, why?

Reply:
I was getting errors on github actions with packemon v3.3.1 dependency boost/cli depending on react 16 / 17.

This is more of a temp fix and we need to upgrade packemon to v4 which uses a version of boost/cli that depends on react 18

Reply:
ok